package.json

```diff
@@ -80,7 +80,7 @@
     "@types/underscore": "1.8.3",
     "@types/xregexp": "^3.0.30",
     "aws-sdk": "^2.155.0",
-    "axe-core": "^3.0.2",
+    "axe-core": "~3.0.2",
```

Comment:
Upgrading axe-core caused more accessibility issues to be detected. I'm going to create a separate issue for this.

Reply:
Instead of switching the `^` to a `~`, could we update the version number to the one we need to be on?
For example, in the yarn.lock, I see the dependency is on version `3.0.3`, but I also see `axe-core@*` is on version `3.5.5`?

Could we just do `yarn add axe-core@3.5.5` so that the number is reflected in the package.json?

Reply:
Unfortunately, `axe-core@3.5.5` is the one showing more accessibility issues. The reason for its existence in the `yarn.lock` file is because `@types/underscore` appears to depend on `underscore@*`.

However, I can upgrade the `axe-core` dependency in `package.json` to `~3.0.3`.